scripts/prepareForFirstWeekLaunch.ts

```diff
@@ -138,97 +217,13 @@ async function main() {
     "(1M SDL/4 Year) lock created at timestamp: ",
     await getCurrentBlockTimestamp(),
   )
-  // set gauge reward to sample tokens
-  console.log(
-    "rewards before desposits: ",
-    await GaugeHelperContract.getGaugeRewards(gauge.address),
-  )
-  await gauge.add_reward(rewardToken1.address, signers[0].address)
-  await gauge.add_reward(rewardToken2.address, signers[0].address)
-  await rewardToken1.approve(gauge.address, MAX_UINT256)
-  await rewardToken2.approve(gauge.address, MAX_UINT256)
-  console.log(
-    "rewardToken1 bal",
-    (await rewardToken1.balanceOf(signers[0].address)).toString(),
-  )
-  console.log(
-    "rewardToken2 bal",
-    (await rewardToken2.balanceOf(signers[0].address)).toString(),
-  )
-  await gauge.deposit_reward_token(
-    rewardToken1.address,
-    BIG_NUMBER_1E18.mul(100_000),
-  )
-  await gauge.deposit_reward_token(
-    rewardToken2.address,
-    BIG_NUMBER_1E18.mul(100_000),
-  )
-  console.log(
-    "rewardToken bal after deposit",
-    (await rewardToken1.balanceOf(signers[0].address)).toString(),
-    (await rewardToken2.balanceOf(signers[0].address)).toString(),
-  )
-  console.log(
-    "gauge bal after deposit",
-    (await rewardToken1.balanceOf(gauge.address)).toString(),
-    (await rewardToken2.balanceOf(gauge.address)).toString(),
-  )
-  console.log(
-    "rewards: ",
-    await GaugeHelperContract.getGaugeRewards(gauge.address),
-  )
+
   // Deposit into gauge with boosted account
+  console.log("lp bal of gauge: ", (await lpToken.balanceOf(gauge.address)).toString())
```

Comment:
Nitpick

This assumption that lp balance of gauge = total amount deposited in the gauge is not true.

This would include any transfers of lp tokens to the gauge, which would not be earning any APYs.

We should be using `gauge.totalSupply()` instead to read number of LP tokens deposited for any reward calculation purposes.